client/verta/verta/_registry/modelversion.py

```diff
@@ -279,7 +294,7 @@ def del_model(self):
         self._update(self._msg, method="PUT")
 
 
-    def log_artifact(self, key, artifact, overwrite=False, extension=None):
+    def log_artifact(self, key, artifact, overwrite=False, _extension=None):
```

Comment:
I'm thinking this shouldn't have been added as a public param in the first place 😞